projects/igniteui-angular/src/lib/chips/chips-area.spec.ts

```diff
@@ -93,6 +174,576 @@ describe('IgxChipsArea', () => {
         expect(chipArea[0].nativeElement.children[0].innerHTML).toContain('New text');
     });
 
+    it('should not be able to drag and drop when chip is not draggable', async(() => {
+        const fix = TestBed.createComponent(TestChipComponent);
+        fix.detectChanges();
+
+        const chipArea = fix.debugElement.queryAll(By.directive(IgxChipsAreaComponent));
+        const chipComponents = chipArea[0].queryAll(By.directive(IgxChipComponent));
+        const firstChip = chipComponents[0].componentInstance;
+        const firstChipElem = firstChip.chipArea.nativeElement;
+
+        const startingTop = firstChipElem.getBoundingClientRect().top;
+        const startingLeft = firstChipElem.getBoundingClientRect().left;
+        const startingBottom = firstChipElem.getBoundingClientRect().bottom;
+        const startingRight = firstChipElem.getBoundingClientRect().right;
+
+        const startingX = (startingLeft + startingRight) / 2;
+        const startingY = (startingTop + startingBottom) / 2;
+
+        simulatePointerEvent('pointerdown', firstChipElem, startingX, startingY);
+        fix.detectChanges();
+
+        fix.whenStable().then(() => {
+            fix.detectChanges();
+            simulatePointerEvent('pointermove', firstChipElem, startingX + 10, startingY + 10);
+
+            return fix.whenStable();
+        }).then(() => {
+            fix.detectChanges();
+            const dragDir = firstChip.dragDir['_dragGhost'];
+
+            expect(dragDir).toBeUndefined();
+        });
+    }));
+
+    it('should be able to drag and drop when chip is draggable', async(() => {
```

Comment:
From the looks the test only drags a chip, so no need to have drop in the title.